src/index.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import _ from 'lodash';
+/* eslint-enable */
+import './style.css';
+
+const todoList = document.querySelector('.items');
+const todos = [
+  {
+    description: 'Take a walk ',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Read a book',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Wash the car',
+    completed: false,
+    index: 0,
+  },
+];
```

Comment:
- [ ] You've implemented your javascript well. however, I'd like you to notice that the requirements need the task to show up in order of the index. when I check your website the first task is "Take a walk" when it should really be "Wash the car" because that has index 0. Please make this change. you can sort the `todos` before displaying them